src/glog/logging.h.in

```diff
@@ -36,6 +36,10 @@
 #ifndef _LOGGING_H_
 #define _LOGGING_H_
 
+#ifdef HAVE_CXX11_CHRONO
```

Comment:
`HAVE_CXX11_CHRONO` is undefined here. See the `ac_` style placeholders how to use optional features correctly.

Reply:
So is adding the required lines in `config.h.cmake.in` not the right way to do this? It seems like the rest of `HAVE_CXX11...` macros do that.

Reply:
`config.h.cmake.in` defines are used only internally. If you want to export the feature availability, you must use `ac_` placeholders that are replaced during configuration time.

Reply:
Got it. Changed it in cmake, but I'll also need to address it in bazel it seems but first I have to learn bazel.

Reply:
Bazel support should be straightforward if you add the placeholders to one of these config blocks:
https://github.com/google/glog/blob/6e46c125e3718ce64570883e2ca558b3c853a125/bazel/glog.bzl#L190-L226

Reply:
I figured it would be that simple but I wasn't quite sure if `ac_cxx11_constexpr` and `ac_cxx11_have_chrono` should always be `1` since they're really dependent on c++11 or greater.

Reply:
In contrast to CMake, Bazel defaults to C++11.